src/Common/Admin/Conditional_Content/Datetime_Conditional_Abstract.php

```diff
@@ -1,73 +1,78 @@
 <?php
-namespace Tribe\Admin\Conditional_Content;
+
+namespace TEC\Common\Admin\Conditional_Content;
 
 use Tribe__Date_Utils as Dates;
+use Tribe__Template as Template;
+use Tribe\Utils\Date_I18n;
 
 /**
  * Abstract class for conditional content.
  *
- * @since 4.14.7
+ * @since TBD
  */
 abstract class Datetime_Conditional_Abstract {
```

Comment:
Aren't changes on this file breaking backwards compatibility ? e.g. ET might be extending this class and not have a delcared return type for method get_start_time being overwritten.

By the way this looks like it could be extending a Provider. The registration of Black_Friday is like just registering a provider with extra steps.

Reply:
Potentially, I will add back the older ones in the deprecated folder.
I've tested with all the other plugins and looked for it, there isn't anything on our codebase. But there might be something external. I am aggressively trying to reduce codebase haha.